### PR TITLE
luxio: init at 13

### PR DESCRIPTION
**Pull request by richardipsum**
###### Motivation for this change

Hi!

I would like to package a git network auth package known as [Gitano](https://wiki.gitano.org.uk/)
Luxio is a dependency of Gitano's, it is a set of lua bindings to POSIX functionality.

I'll be opening separate PRs for each Gitano dependency, since that should make it easier to review.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [] NixOS
   - [ ] macOS
   - [X] Linux

I have tested all Gitano dependencies by building and testing Gitano itself using Nix on GNU/Linux.
I'm also happy to maintain Gitano and all its dependencies for the foreseeable future.

Thanks!
Richard

### CODE REVIEW

File: pkgs/top-level/lua-packages.nix [hunk 1]
```diff
@@ -270,6 +270,32 @@ let
...
+      sha256 = "1hvwslc25q7k82rxk461zr1a2041nxg7sn3sw3w0y5jxf0giz2pz";
+    };
+    nativeBuildInputs = [ which pkgconfig ];
+    postPatch = ''
+      patchShebangs .
+    '';
+    meta = {
+      platforms = stdenv.lib.platforms.unix;
+      license = stdenv.lib.licenses.mit;
+      description = "Lightweight UNIX I/O and POSIX binding for Lua";
```
Comment (Mic92) [inline pkgs/top-level/lua-packages.nix]: Do you want to maintain this package?

Comment (richardipsum) [reply pkgs/top-level/lua-packages.nix]: Thanks for such a quick response! And also for fixing this up. I'll try and test subsequent PRs more thoroughly.

But yes I'm happy to be the maintainer for this and all related packages. :smiley:

Comment (Mic92) [reply pkgs/top-level/lua-packages.nix]: Ok then please add yourself to `lib/maintainers.nix` as well as to `meta.maintainers` field of this package.

Comment (richardipsum) [reply pkgs/top-level/lua-packages.nix]: Done

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**